src/API/Management/Organizations.php

```diff
@@ -0,0 +1,598 @@
+     */
+    public function getAll(
+        array $params = []
+    )
+    {
+        return $this->apiClient->method('get')
+            ->addPath('organizations')
+            ->withDictParams($this->normalizeRequest($params))
+            ->call();
+    }
+
+    /**
+     * Get details about an organization, queried by it's ID.
+     * Required scope: "read:organizations"
+     *
+     * @param string $organization Organization (by ID) to retrieve details for.
+     *
+     * @return mixed
+     *
+     * @throws RequestException When API request fails. Reason for failure provided in exception message.
+     */
+    public function get(
+        string $organization
+    )
+    {
+        $this->checkEmptyOrInvalidString($organization, 'organization');
+
+        return $this->apiClient->method('get')
+            ->addPath('organizations', $organization)
+            ->call();
+    }
+
+    /**
+     * Get details about an organization, queried by it's `name`.
+     * Required scope: "read:organizations"
+     *
+     * @param string $organizationName Organization (by name parameter provided during creation) to retrieve details for.
+     *
+     * @return mixed
+     *
+     * @throws RequestException When API request fails. Reason for failure provided in exception message.
+     */
+    public function getByName(
+        string $organizationName
+    )
+    {
+        $this->checkEmptyOrInvalidString($organizationName, 'organizationName');
+
+        return $this->apiClient->method('get')
+            ->addPath('organizations', 'name', $organizationName)
+            ->call();
+    }
+
+    /**
+     * List the enabled connections associated with an organization.
+     * Required scope: "read:organization_connections"
+     *
+     * @param string              $organization Organization (by ID) to list connections of.
+     * @param array<string,mixed> $params       Optional. Additional options to include with the request, such as pagination or filtering parameters.
+     *
+     * @return mixed
+     *
+     * @throws RequestException When API request fails. Reason for failure provided in exception message.
+     */
+    public function getEnabledConnections(
+        string $organization,
+        array $params = []
+    )
+    {
+        $this->checkEmptyOrInvalidString($organization, 'organization');
+
+        return $this->apiClient->method('get')
+            ->addPath('organizations', $organization, 'enabled_connections')
+            ->withDictParams($this->normalizeRequest($params))
+            ->call();
+    }
+
+    /**
+     * Get a connection (by ID) associated with an organization.
+     * Required scope: "read:organization_connections"
+     *
+     * @param string              $organization Organization (by ID) to list connections of.
+     * @param array<string,mixed> $params       Optional. Additional options to include with the request, such as pagination or filtering parameters.
```

Comment:
Should this instead refer to `connection` rather than `params`? Looks like this is for another method.

Reply:
Whoops, good catch! Fixed 👍